test/elixometer_test.exs

```diff
@@ -70,6 +70,12 @@ defmodule ElixometerTest do
     :timer.sleep(50)
   end
 
+  # Remove the :ms_since_reset entries from a list of metric values because
+  # they introduce non-determinism within test runs.
+  defp without_ms_since_reset(values) do
+    Enum.reduce(values, [], fn {key, [value, _ms]}, acc -> [acc | {key, value}] end)
```

Comment:
Feel like we could make intent clearer with something like Enum.map(values, fn {key, list} -> {key, Keyword.delete(list, :ms_since_reset)} end)

Reply:
That's a nice improvement. Thanks for the idea!